index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="header">
+        <nav class="navbar">
+            <label class="mylogo">RASHID</label>
+            
+            <i  class="menu-icon fa-solid fa-bars"></i>
+        </nav>
+        <h1 class="heading">I’m Rashid Menhas
+            Glad to see you!</h1>
+        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h5 class="btn-connect">LET’S CONNECT</h5>
```

Comment:
- [ ] The heading is perfectly fine the only thing left is font-size So I would suggest you increase the size so It fit the content. let's increase it and make font-size: 19px;